Packaging Error in 7.6.1

Getting with v7.6.1 following error:

```
components/network/vis-network.service.ts:16:8 - error TS7016: Could not find a declaration file for module 'vis-network'. '/Users/mreinhardt/git/private/ngx-vis/node_modules/vis-network/peer/umd/vis-network.js' implicitly has an 'any' type.
  Try `npm install @types/vis-network` if it exists or add a new declaration (.d.ts) file containing `declare module 'vis-network';`
```

Beginning with 7.4.x this error appeared...

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -1,0 +1,1 @@
+export * from "./index-legacy-bundle";

